README.md

```diff
@@ -8,11 +8,11 @@
 
 A small Java library for talking to GitHub/GitHub Enterprise and interacting with projects.
 
-It supports authentication via simple access tokens, JWT endpoints and GitHub Apps (via private key).
+It supports authentication via simple access tokens, JWT endpoints, and GitHub Apps (via private key).
```

Comment:
Usually, the comma you added is not needed (I am not sure whether correct grammar even accepts this comma). I only add this comma if it increases the readability if I list long items. Think of `very long item, another very long item, and a third long item`. In the above case, I would not add it.